apps/api/src/app/workflows-v2/util/template-parser/parser-types.ts

```diff
@@ -0,0 +1,10 @@
+export type InvalidVariable = {
```

Comment:
Consider moving these types into the liquide-parse.ts file to save us from yet another file.

Reply:
No problem, i will merge it i wanted to split the logic as much as possible.